scripts/extract_schema.py

```diff
@@ -0,0 +1,88 @@
+# !/usr/bin/env python
```

Comment:
nit: Remove the copyright for new files (CC: @w4nderlust), and add docstrings to each function.

Reply:
done